packages/flutter_tools/lib/src/isolated/native_assets/native_assets.dart

```diff
@@ -30,11 +29,159 @@ import 'macos/native_assets.dart';
 import 'macos/native_assets_host.dart';
 import 'windows/native_assets.dart';
 
+/// The assets produced by a Dart build and the dependencies of those assets.
+///
+/// If any of the dependencies change, then the Dart build should be performed
+/// again.
+final class DartBuildResult {
+  const DartBuildResult(this.codeAssets, this.dependencies);
```

Comment:
Nit: prefer required named arguments.

Reply:
Done.

Though to be honest I think it makes the code worse: This class is only used internally within flutter tools. Making them named will just increase code size and reduce information density:

e.g. What used to be
```dart
return DartBuildResult(codeAssets, dependencies);
```
becomes now
```dart
return DartBuildResult(codeAssets: codeAssets, dependencies: dependencies);
```
why is the second one better than the first?